talib/abstract.py

```diff
@@ -0,0 +1,34 @@
+from ._ta_lib import Function as _Function, __TA_FUNCTION_NAMES__
+
+__ABSTRACT_FUNCTION_NAMES__ = ["_get_defaults_and_docs"]
+
+_ta_func = __import__(
```

Comment:
Yes.That's not necessary. I've changed this line and some other similar lines to `import talib._ta_lib as _ta_lib`.